src/module/Entity/src/Entity/Form/ModulePageForm.php

```diff
@@ -32,7 +32,7 @@ function __construct(LicenseInterface $license)
         $select->setLabel('Select an icon');
         $select->setAttribute('id', 'icon');
         $select->setValueOptions(array(
-                '' => 'Explanation',
+                'none' => 'Explanation',
```

Comment:
Would be possible too, should I change it? In frontend this key is used for selecting the fontawesome icon, so the names of those icons is used here. But explanation should not have an icon, hence "none". But "explanation" would work fine too.